completions/quick-lint-js.bash

```diff
@@ -0,0 +1,39 @@
+#/usr/bin/env bash
+
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+_quick-lint-js () {
+        local cur prev opts
+
+        COMPREPLY=()
+        cur="${COMP_WORDS[COMP_CWORD]}"
+        prev="${COMP_WORDS[COMP_CWORD-1]}"
+        opts="-h --help -v --version --lsp --lsp-server --exit-fail-on --output-format --vim-file-bufnr"
```

Comment:
Is it possible to include a `=` after `--exit-fail-on` and `--vim-file-bufnr` and `--output-format`?